add-chain/testdata/monorepo/deployments/.chainId

```diff
@@ -0,0 +1 @@
+11155111
```

Comment:
There is a lot of data committed in this folder, most of which I think we don't need. We only need the address from each deployment. 

I suggest we commit test data in the newer deployment artifact system, see https://github.com/ethereum-optimism/superchain-registry/pull/200 . It's just a single json file with a list of addresses. This will save us a a lot of lines.

If we really want to test agasint legacy deployment artifacts formats, we can strip the artifacts down so they only contain the information we are looking for.

Reply:
`genesis.json` is also quite a huge file -- it is in the `testdata` folder, so are we sure this won't mean downstream dependencies need to download it?

Another issue is that this file is also in a format which is constantly changing. See this comment https://github.com/ethereum-optimism/optimism/pull/10231#discussion_r1577627727 .

Reply:
Removed individual contract json files from `testdata` and are now reading contract addresses from `.deploy` file instead: https://github.com/ethereum-optimism/superchain-registry/pull/204/commits/61ea05f6243255f92b1fac7539c84666d6b7dd29

Reply:
`genesis.json` is actually not used by the `add-chain` module. Rather, it is needed for the `registry-data` module. Removed that file in this commit: https://github.com/ethereum-optimism/superchain-registry/pull/204/commits/c331023f015b721710d745e8c55f945271cf7c93